server/src/main/java/org/opensearch/rest/action/admin/cluster/RestClusterHealthAction.java

```diff
@@ -94,6 +94,26 @@ public static ClusterHealthRequest fromRequest(final RestRequest request) {
         );
         parseDeprecatedMasterTimeoutParameter(clusterHealthRequest, request, deprecationLogger, "cluster_health");
         clusterHealthRequest.timeout(request.paramAsTime("timeout", clusterHealthRequest.timeout()));
+        String awarenessAttribute = request.param("awareness_attribute");
+        if (awarenessAttribute != null) {
+            clusterHealthRequest.setAwarenessAttribute(awarenessAttribute);
+        }
+        String level = request.param("level");
```

Comment:
I am assuming this existed before how was level being set all his while?

Reply:
It was only checked in response previously but for validation standpoint we need to check in request class as well so added the same